### DISCUSSION TITLE
What is missing from this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#545] [Galaxy Trivia] [answered]
`teloiv ogidni eulb ==?== wolley egnaro der`

**Comment by peterbaccaro** [comment]
**neerg** as these are the colours of the rainbow reversed.

**Comment by ParanoidUser** [comment]
@Veercodeprog, @juletopi, @ErKiran want to give it a shot? 😼

**Comment by juletopi** [comment]
It's **neerg**. As in: "`teloiv ogidni eulb ==?== wolley egnaro der`"

`teloiv ogidni eulb neerg wolley egnaro der`
&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199;&#8199; =
`red orange yellow green blue indigo violet`

**Comment by ErKiran** [comment]
esolc

So the correct sentence is: "Close the ==?== door and ignore the open yellow towel."

**Comment by Veercodeprog** [comment]
The missing item in this sequence is the word "gnirts" (which is "string" spelled backwards), so the complete sequence is:

"teloiv ogidni eulb gnirts wolley egnaro der"

**Comment by thuongtruong1009** [comment, marked as answer]
The given sequence "teloiv ogidni eulb ==?== wolley egnaro der" appears to be the reversed form of "red orange yellow ==?== blue indigo violet".

Therefore, the missing word in the sequence should be "green".

Hence, the complete sequence should be "red orange yellow green blue indigo violet" which represents the colors of a rainbow in order.
